ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -645,6 +646,18 @@ match_list_input:
           match_regex: < match string >
 ```
 
+#### AS Paths
+
+```yaml
+as_path:
+  regex_mode: < asn | string >
+  access_lists:
```

Comment:
Since this is a new data model, we should use list of dicts instead. 
Also you can have multiple entries in the same list.
```eos
ip as-path access-list test permit 123 any
ip as-path access-list test permit 321 any
```
My suggestion:
```yaml
as_path:
  regex_mode: < asn | string >
  access_lists:
    - name: < access_list_name_1 >
      entries:
        - type: < permit | deny >
          match: "< regex to match >"
          origin: < any | egp | igp | incomplete >
```